src/components/sections/ProductGallery/useDelayedFacets.ts

```diff
@@ -9,7 +9,7 @@ export const useDelayedFacets = (data?: ProductGalleryQueryQuery) => {
 
   return useMemo(() => {
     if (data) {
-      facets.current = data.search.facets
+      facets.current = data.search.facets as any
```

Comment:
👀

Reply:
this is needed, because the husky screams at this error during the commit.